File: app/src/main/java/forpdateam/ru/forpda/views/adapters/BaseSectionedViewHolder.java
package forpdateam.ru.forpda.views.adapters;

import android.view.View;

import com.afollestad.sectionedrecyclerview.SectionedViewHolder;

/**
 * Created by radiationx on 14.09.17.
 */

public class BaseSectionedViewHolder<T> extends SectionedViewHolder {
    public BaseSectionedViewHolder(View itemView) {
        super(itemView);
    }


    public void bind(T item, int section, int relativePosition, int absolutePosition) {
    }

    public void bind(T item, int section) {
    }

    public void bind(T item) {
    }

    public void bind(int section) {
    }

    public void bind() {
    }
}

File: app/src/main/java/forpdateam/ru/forpda/ui/views/adapters/BaseSectionedViewHolder.kt
package forpdateam.ru.forpda.ui.views.adapters

import android.view.View
import com.afollestad.sectionedrecyclerview.SectionedViewHolder

/**
 * Created by radiationx on 14.09.17.
 */
open class BaseSectionedViewHolder<T>(itemView: View) : SectionedViewHolder(itemView) {
    open fun bind(item: T, section: Int, relativePosition: Int, absolutePosition: Int) {
    }

    fun bind(item: T, section: Int) {
    }

    open fun bind(item: T) {
    }

    open fun bind(section: Int) {
    }

    fun bind() {
    }
}
